tests/ui/offset-of/offset-of-arg-count.stderr

```diff
@@ -0,0 +1,20 @@
+error: expected one of `!`, `(`, `+`, `,`, `::`, or `<`, found `<eof>`
+  --> $DIR/offset-of-arg-count.rs:6:16
+   |
+LL |     offset_of!(NotEnoughArguments);
+   |                ^^^^^^^^^^^^^^^^^^ expected one of `!`, `(`, `+`, `,`, `::`, or `<`
+
+error: expected 2 arguments
+  --> $DIR/offset-of-arg-count.rs:7:5
+   |
+LL |     offset_of!(NotEnoughArgumentsWithAComma, );
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: expected 2 arguments
+  --> $DIR/offset-of-arg-count.rs:8:5
+   |
+LL |     offset_of!(Container, field, too many arguments);
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: aborting due to 3 previous errors
```

Comment:
Not necessarily a blocker, but these errors can be improved a lot, as of now they are somewhat confusing.

Reply:
You are right but IMO it's done in a future PR. People want to extend the feature but for that it would be best merged to master. Also the PR had to be rebased already. So IMO it would be better to merge it more quickly and then look at the imperfections later: that's what the unstable period of a feature is for.

Reply:
Yes, I agree, just wanted to write that down, for future reference.